libbeat/dashboards/import_dashboards.go

```diff
@@ -448,16 +448,15 @@ func (imp Importer) ImportDir(dirType string, dir string) error {
 	// check if the directory exists
 	if _, err := os.Stat(dir); err != nil {
 		// nothing to import
-		fmt.Println("No directory", dir)
-		return nil
+		return fmt.Errorf("No directory %s", dir)
```

Comment:
Oh, I see, this isn't the `_meta` dir, it's another directory further down the tree. I'll update this PR to account for this and the other non-error error.